dts/bindings/gpio/ti,tca9538.yaml

```diff
@@ -24,6 +24,17 @@ properties:
       Connection for the NINT signal. This signal is active-low when
       produced by tca9538 GPIO node.
 
+  input-latch:
+    type: int
+    description: |
+      Setting property for Input Latch.
+
+  interrupt-mask:
+    type: int
+    description: |
+      Setting property for Interrupt mask.
+
+
```

Comment:
Please improve the descriptions here. It's difficult to know what to set these properties to.

Reply:
Has been updated!